.github/workflows/tests.yml

```
@@ -38,7 +38,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.7, 3.8, 3.9]
+        python: [3.7, 3.8, 3.9, 3.10]
```

You can use `3.10-dev` in quotes to solve the version issue here.
```suggestion
        python: [3.7, 3.8, 3.9, '3.10-dev']
```